doc/src/manual/linear-algebra.md

```diff
@@ -221,6 +221,39 @@ operators are generic and match the other matrix in the binary operations [`+`](
 with the identity operator `I` is a noop (except for checking that the scaling factor is one)
 and therefore almost without overhead.
 
+To see the `UniformScaling` operator in action:
+
+```jldoctest
+julia> ustwo = UniformScaling(2);
```

Comment:
Call it something else? Perhaps just `U` or something, took me a while to decipher what `ustwo` meant :)

Reply:
sure, good idea :)